### ISSUE TITLE
MobileNet model is supported

### ISSUE THREAD

**Issue by steven-ja** [HSE-asavchenko/hsemotion#3]
Hello everyone, thanks for this great work!

In your [face emotion recognition repository](https://github.com/HSE-asavchenko/face-emotion-recognition), there is also MobileNet model weights. 
However, in this repository it is not supported as *model_name*. This is a pity since is the most efficient model if we consider the inference time vs. accuracy.

Can it be added in some way?

Thanks

**Comment by av-savchenko** [comment]
Hello! Thanks for your interest!
MobileNet is very different from other models because MobileNet was trained in the TensorFlow, while EfficientNet models were trained in PyTorch. I decided not to add Tensorflow's dependencies into hsemotion library, and I'm not sure that I will have time to train MobileNet in PyTorch soon. The only one workaround now is to use the MobileNet model in your code similarly to [display_emotions.ipynb](https://github.com/HSE-asavchenko/face-emotion-recognition/blob/main/src/display_emotions.ipynb)
